src/vue.d.ts

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
```

Comment:
Remove this import as it's not used.

Reply:
if you remove it you will get this error:

![image](https://user-images.githubusercontent.com/6597815/130227448-5cd4ec0c-70ef-426d-82ba-7725ed8924bf.png)